### PR TITLE
Fix unzip of config files and incorrect arg name in docker-compose

### PR DESCRIPTION
**Pull request by SvenMarcus**

### CODE REVIEW

**Comment by SvenMarcus** [comment]
The last commit also fixes a bug with the `OCRD_MANAGER_IP` lookup in `startup.sh` by executing the command in a sub shell

File: kitodo/Dockerfile [hunk 1]
```diff
@@ -54,11 +54,11 @@ RUN mkdir /.ssh && \
...
 ADD https://raw.githubusercontent.com/vishnubob/wait-for-it/master/wait-for-it.sh /tmp/wait-for-it.sh
 
-RUN	unzip /tmp/kitodo/kitodo-config-modules.zip -x kitodo-config-modules/scripts/*.bat -d /tmp/kitodo/kitodo-config-modules && \
+RUN	unzip /tmp/kitodo/kitodo-config-modules.zip -x *.bat -d /tmp/kitodo/ && \
 	rm -f /tmp/kitodo/kitodo-config-modules.zip && \
 	chmod +x /tmp/wait-for-it.sh && \
-	dos2unix /tmp/kitodo/kitodo-config-modules/scripts/*.sh && \
-	chmod 544 /tmp/kitodo/kitodo-config-modules/scripts/*.sh
+	dos2unix /tmp/kitodo/kitodo_3-4-2_config_modules/scripts/*.sh && \
+	chmod 544 /tmp/kitodo/kitodo_3-4-2_config_modules/scripts/*.sh
```
Comment (markusweigelt) [inline kitodo/Dockerfile:62]: For this case i want to use a seperate build variable so i changed dir name to more general .

File: kitodo/startup.sh [hunk 2]
```diff
@@ -11,7 +11,7 @@ chmod go-rwx /.ssh/*
 if [ -n "$OCRD_MANAGER" ]; then
     OCRD_MANAGER_HOST=${OCRD_MANAGER%:*}
     OCRD_MANAGER_PORT=${OCRD_MANAGER#*:}
-	OCRD_MANAGER_IP=nslookup $OCRD_MANAGER_HOST | grep 'Address\:' | awk 'NR==2 {print $2}'
+	OCRD_MANAGER_IP=$(nslookup $OCRD_MANAGER_HOST | grep 'Address\:' | awk 'NR==2 {print $2}')
```
Comment (markusweigelt) [inline kitodo/startup.sh:14]: Interesting for me to know: why is it necessary to use a subshell here. We are not running into a "race problem" here?

**Review by markusweigelt — APPROVED** [review]

File: .gitignore [hunk 3]
```diff
@@ -1,2 +1,3 @@
 **/projects
-**/build-resources
\ No newline at end of file
+**/build-resources
+**/config_modules
```
Comment (markusweigelt) [inline .gitignore:3]: Please add the newline.

**Review by markusweigelt — CHANGES_REQUESTED** [review]
After the tiny improvements i will rebased and merge this.

**Review by markusweigelt — APPROVED** [review]

**Pull request merged**